### ISSUE TITLE
Planar source defects

### ISSUE THREAD

**Issue by zubenscott** [fangq/mcx#29]
When using the 'planar' source with mcxlab I receive a non-uniform planar illumination. Here's a plot of the normalized fluence on the surface of a phantom close to the source:

![fluencedefects](https://user-images.githubusercontent.com/27812066/29207583-975ea3a8-7e7e-11e7-99a8-4518b9c605f1.png)

For various parameter values, I get the same 4 defects. Changing photon number, size and shape of the planar source don't appear to make them go away, nor does adding a focal length for the source.  Here's the code I use,

### First the setup of the phantom:

```
unitinmm=.2; dim = (1/unitinmm)*[60,40, 30];
volume_center = [dim(1)/2+1;dim(2)/2+1;dim(3)/2+1];
% Defining simple rectangular phantom
object_data=uint8(ones(dim(1),dim(2), dim(3)));
object_props=[0.00001 0.0 0.9 1.33;0.003 5 0.9 1.33];
fluence=illuminate_PreXion(object_data,object_props,volume_center,unitinmm);
```

### Matlab function that defines the illuminating geometry and calls mcx:

[illuminate_PreXion.txt](https://github.com/fangq/mcx/files/1217706/illuminate_PreXion.txt)
(save as .m file to run)

**Comment by fangq** [comment]
@zubenscott, sorry for my slow response. I just tested your script on the latest git code (nightly-build), and I do not see these artifacts. I am wondering if you can download the latest mcxlab (http://mcx.space/nightly/) and verify if this issue still present? 

my screen capture is attached, generated using 5e8 photons.

![planar_source_artifact](https://user-images.githubusercontent.com/226913/34854483-f8c285d6-f707-11e7-87ba-62084b503a02.png)

**Comment by fangq** [comment]
close, no longer reproducible. if the issue remains under certain conditions, feel free to reopen.

**Comment by fangq** [comment]
this issue seems to be caused by the same problem that was fixed in Issue #31. 
it is also similar to the one reported in Issue #26. 

glad this was fixed. Please download the latest nightly build for Windows http://mcx.space/nightly/win64/
